ast/ast.go

```diff
@@ -289,15 +289,20 @@ type (
 
 	PutProc struct {
 		Node
-		Clauses []Assignment `json:"clauses"`
+		Clauses []ExpressionAssignment `json:"clauses"`
 	}
 )
 
-type Assignment struct {
+type ExpressionAssignment struct {
 	Target string     `json:"target"`
 	Expr   Expression `json:"expression"`
 }
 
+type FieldAssignment struct {
+	Target string `json:"target"`
```

Comment:
Out of curiosity, and I know this change was done a while ago, but why the move away using a `string` here instead of `FieldExpr`?

Reply:
It was done here: https://github.com/brimsec/zq/pull/699/